torch/csrc/jit/frontend/sugared_value.cpp

```diff
@@ -108,18 +108,33 @@ std::shared_ptr<SugaredValue> SimpleValue::attr(
        }},
       {TypeKind::TensorType,
        {
-           {"dtype", "prim"},         {"device", "prim"},
-           {"grad", "prim"},          {"data", "prim"},
-           {"shape", "prim"},         {"is_cuda", "prim"},
-           {"is_xpu", "prim"},        {"is_sparse", "prim"},
-           {"is_sparse_csr", "prim"}, {"is_mkldnn", "prim"},
-           {"is_mlc", "prim"},        {"is_quantized", "prim"},
-           {"is_vulkan", "prim"},     {"is_meta", "prim"},
-           {"is_leaf", "aten"},       {"requires_grad", "prim"},
-           {"layout", "prim"},        {"T", "prim"},
-           {"ndim", "prim"},          {"name", "prim"},
-           {"real", "aten"},          {"imag", "aten"},
-           {"retains_grad", "aten"},  {"is_ort", "prim"},
+           {"dtype", "prim"},
+           {"device", "prim"},
+           {"grad", "prim"},
+           {"data", "prim"},
+           {"shape", "prim"},
+           {"is_cuda", "prim"},
+           {"is_xpu", "prim"},
+           {"is_sparse", "prim"},
+           {"is_sparse_csr", "prim"},
+           {"is_mkldnn", "prim"},
+           {"is_mlc", "prim"},
+           {"is_quantized", "prim"},
+           {"is_vulkan", "prim"},
+           {"is_meta", "prim"},
+           {"is_leaf", "aten"},
+           {"requires_grad", "prim"},
+           {"layout", "prim"},
+           {"T", "prim"},
+           {"H", "prim"},
+           {"mT", "aten"},
+           {"mH", "aten"},
```

Comment:
@tugsbayasgalan Is this correct? I did not know whether they should be `"prim"` or `"aten"` or what really.

Reply:
I don;t think it matters. But i think prim would be better

Reply:
As per https://github.com/pytorch/pytorch/blob/50edc2679d53b38ab4fc8ab684cd0f8f50df8358/aten/src/ATen/core/interned_strings.h#L502-L505
I believe that it's correct as it stands, as `at::mT` and `at::mH` do exist, while `at::T` and `at::H` do not (they are called `at::numpy_T` and `at::matrix_H`).
Is this correct?

Reply:
So I discussed with @eellison and we concluded that it doesn't really matter what the namespace is. But I said prim is better because these newly added operator names can collide with the entries in native_functions.yaml. So I think maybe they all should be prim? @eellison let us know if it is wrong